sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.1.0-beta.2 (Unreleased)
```

Comment:
Don't need to increase the version here, please just use the beta.1 which is reserved for next release

Reply:
OK will do

Reply:
Done